### ISSUE TITLE
 '_AxesStack' object is not callable

### ISSUE THREAD

**Issue by C0casio45** [networkx/networkx#5798]
### Current Behavior
When I'm running networkx on jupyterlab hosted on a arm architecture (raspberrypi 4 8go) I got this error:

```
File /usr/local/lib/python3.9/site-packages/networkx/drawing/nx_pylab.py:113, in draw(G, pos, ax, **kwds)
    111 cf.set_facecolor("w")
    112 if ax is None:
--> 113     if cf._axstack() is None:
    114         ax = cf.add_axes((0, 0, 1, 1))
    115     else:

TypeError: '_AxesStack' object is not callable
```

### Expected Behavior
I used this example to be sure of my code
[networkx example](https://networkx.org/documentation/networkx-2.3/auto_examples/basic/plot_properties.html#sphx-glr-auto-examples-basic-plot-properties-py)

### Steps to Reproduce
Use docker and install networkx by this way
`RUN pip3 install --no-cache-dir networkx numpy ipywidgets`

### Environment
Python version: 3.9
NetworkX version: 2.8.4
Raspbian 10
Docker 20.10.17
Jupyterlab 3.4.3

**Comment by MridulS** [comment]
I can't really reproduce this (don't have a raspberrypi). But this error is coming from matplotlib. What version of `matplotlib` do you have?

Also the link to the example is a old version (which has a typo), pleas use the latest stable release https://networkx.org/documentation/stable/auto_examples/basic/plot_properties.html

**Comment by C0casio45** [comment]
matplotlib 3.6.0

**Comment by C0casio45** [comment]
huh it's strange :/ I got a dev version not referenced on their github

**Comment by MridulS** [comment]
Well what's happening after this clone?
Also I would suggest installing the "default" requirements of networkx. You can change your RUN command to:
```
pip3 install --no-cache-dir networkx[default] ipywidgets
```

**Comment by C0casio45** [comment]
Ok i reinstalled matplotlib with version 3.5.2 hardwritten in the git clone and everything works
I dunno why I got a dev version who is not released :/

**Comment by QuLogic** [comment]
This is still a bug that `networkx` needs to fix; it is calling private API that is changed in 3.6.0rc1, and [fails to run](https://copr.fedorainfracloud.org/coprs/qulogic/matplotlib-3.6.0/build/4751432/):
```
Extension error:
Here is a summary of the problems encountered when running the examples

Unexpected failing examples:
/builddir/build/BUILD/networkx-networkx-2.8.5/examples/basic/plot_properties.py failed leaving traceback:
Traceback (most recent call last):
  File "/builddir/build/BUILD/networkx-networkx-2.8.5/examples/basic/plot_properties.py", line 48, in <module>
    nx.draw(G, pos=pos, with_labels=True)
  File "/builddir/build/BUILD/networkx-networkx-2.8.5/build/lib/networkx/drawing/nx_pylab.py", line 113, in draw
    if cf._axstack() is None:
       ^^^^^^^^^^^^^
TypeError: '_AxesStack' object is not callable
```

**Comment by rossbar** [comment]
This should be fixed by #5938 , thanks for the ping @QuLogic !
